lib/buffer-builder.ts

```diff
@@ -150,6 +150,16 @@ export class BufferBuilder {
     return this.buffer.flush();
   }
 
+  /**
+   * Register Paper Cut Action
+   * @return void
+   */
+  public paperCut(): BufferBuilder {
+    this.buffer.write( Command.GS_v(1) );
```

Comment:
Please, remove the white-spaces inside ( );

Reply:
ok